### PR TITLE
Add instructions for building Rinvid on Windows 10

### PR DESCRIPTION
**Pull request by Seviel**
NOTE: Not to be merged before #52

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +19,45 @@ After that, clone the repo and inside any repo directory run:
...
+Copy contents of folder where your MinGW installed into <msys2_install_path>/mingw64  
+
+#### Step 5
+Add <msys2_install_path>/mingw64/bin to Path environment variable, and remove original MinGW installation location from Path env var.  
+
+#### Step 6 (Only if building SFML from source)
+Install CMake from this [link](https://cmake.org/download/)  
+
+#### Step 7
+Download SFML from this [page](https://www.sfml-dev.org/download/sfml/2.5.1/). If you're building SFML from source, follow instructions from this [link](https://www.sfml-dev.org/tutorials/2.5/compile-with-cmake.php)
```
Comment (zolalsl) [inline README.md:51]: Seems like SFML/Window.hpp can't be found. Are we sure this is right location for SFML headers?

Comment (Seviel) [reply README.md:51]: Apparently, location of standard compiler includes can vary. I'll update the manual.

Comment (zolalsl) [reply README.md:51]: Yeah. Seems like it's enough to find which `stdio` was being loaded and just place SFML folder there. 

Also seems like windows needs to be in "developer mode".

**Review by zolalsl — APPROVED** [review]
LGTM

**Pull request merged**